.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: generic
+sudo: required
+
+services:
+  - docker
+
+before_install:
+  - docker build -t iot-lab-gateway .
+  - docker build -t iot-lab-gateway-test tests
+
+script:
+  - docker run -v gateway_code:/home/iot-lab-gateway/gateway_code -v $CWD/tests_utils/cfg_dir:/var/local/config -e BOARD_TYPE=m3 -e CONTROL_NODE_TYPE=no -e HOSTNAME=m3-00 -e IOTLAB_GATEWAY_NO_INTEGRATION_TESTS --privileged iot-lab-gateway-test
```

Comment:
Do you think this line could be split (<80 characters long) ?

Reply:
It'd be great. But I havent found a way to do that in a travis yml, yet

Reply:
how about sticking this in a separate docker-test script then, or in a simple pytest ?
and it would give a usage example in the source tree :)

Reply:
script:
  - >
    docker run -v gateway_code:/home/iot-lab-gateway/gateway_code
    -v $CWD/tests_utils/cfg_dir:/var/local/config
    -e BOARD_TYPE=m3
    -e CONTROL_NODE_TYPE=no
    -e HOSTNAME=m3-00 
    -e IOTLAB_GATEWAY_NO_INTEGRATION_TESTS
    --privileged iot-lab-gateway-test

might work, if travis doesnt like it I'll put that in a tests_utils/docker-test